pimpmycause/pimpmycause/templates/core/team.html

```diff
@@ -0,0 +1,50 @@
+{% extends "layout.html" %}
+
+{% load static %}
+
+{% block main %} 
+    <section class="page-about">
+        <div class="content-wrap">
+             <div class="row">
+                <div class="large-12 medium-12 small -12 columns">
+                    <h1 class="pnl__title">Meet The Team</h1>
+
+                    <div class="team">
+
+                        <div class="team-member">
```

Comment:
this is wrong ty! definitely not bem :( will change to `team__person-info` or something better, need to think about it a little.